city_scrapers/spiders/chi_ssa_62.py

```diff
@@ -0,0 +1,119 @@
+from datetime import datetime, time
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE, BOARD, FORUM, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa62Spider(CityScrapersSpider):
+    name = "chi_ssa_62"
+    agency = "Chicago Special Service Area #62 Sauganash"
+    timezone = "America/Chicago"
+    allowed_domains = ["escc60646.com"]
+    start_urls = ["http://escc60646.com/our_events/"]
```

Comment:
On the off chance they start publishing minutes, it might be good to start from `http://escc60646.com/our_events/?date1=all` and only `yield` Meetings starting from 90 days ago